web/olga/analytics/tests/test_views.py

```diff
@@ -132,10 +180,25 @@ def test_logger_debug_occurs(self, mock_uuid4, mock_logger_debug):
         """
         mock_uuid4.return_value = MockUUID4()
 
-        self.client.post('/api/token/registration/', {})
+        x_forward_for = '123.0.0.1'
+        uid = hashlib.md5(x_forward_for).hexdigest()
+        self.client.post('/api/token/registration/', HTTP_X_FORWARDED_FOR=x_forward_for)
 
-        mock_logger_debug.assert_called_once_with(
-            'OLGA registered edX installation with token %s', mock_uuid4.return_value.access_token
+        mock_logger_debug.assert_any_call(
+            'OLGA registered edX installation with token %s for uid %s',
+            mock_uuid4.return_value.access_token,
+            uid
+        )
+
+        x_forward_for = '123.0.0.2'
+        uid = hashlib.md5(x_forward_for).hexdigest()
+        self.client.post('/api/token/registration/', HTTP_X_FORWARDED_FOR=x_forward_for)
+        mock_uuid4.return_value = MockUUID4()
```

Comment:
We've discussed that it would be ok to have only one `mock_uuid4.return_value` definition with the verbose comment

Reply:
MockUUID4 return new random value for each call